scripts/chroot_reconst.sh

```diff
@@ -64,6 +64,9 @@ apt-get -qq -y remove xscreensaver
 # This switch offs the sleep mode
 sudo systemctl mask sleep.target suspend.target hibernate.target hybrid-sleep.target
 
+# This switches off notifications(autostart)
+rm /etc/xdg/autostart/lxqt-notifications.desktop
```

Comment:
Is there no other way to silence that? 

Anyway, this could be an interim fix for the related stub (dummy feature).

Reply:
I am really struggling in finding an alternative to this, 
I have researched on this and deleting the module on autostart comes out to be the least worst method to me as users will be able to config it again using `lxqt-config-notificationd`

Reply:
I will test more on this